gitpod-io/gitpod
# [server] Return NOT_FOUND when workspace does not exist

## Description

Because right now it returns:
```
"jsonrpc2: code -32603 message: Request getWorkspace failed with message: No workspace with id 'some-id' found.
```
which is hard to handle nicely when consuming the API.

## Related Issue(s)

Fixes #

## How to test

1. On preview, run `await window._gp.gitpodService.server.getWorkspace("foo")` in console and observe it returns NOT_FOUND status code

## Release Notes

```release-note
[server] getWorkspace() returns NOT_FOUND when workspace cannot be fetched from db
```

## Documentation

NONE
